benchmarks/htap/htap_loader.py

```diff
@@ -0,0 +1,306 @@
+from datetime import datetime
+from io import StringIO, SEEK_SET
+
+from psycopg2.extras import execute_values
+
+from s64da_benchmark_toolkit.dbconn import DBConn
+
+from benchmarks.htap.lib.helpers import (
+        Random, TPCCText, TPCHText, NATIONS, REGIONS, TimestampGenerator
+)
+
+MAXITEMS = 100000
+DIST_PER_WARE = 10
+CUST_PER_DIST = 3000  # Standard says: 30k customers total (for all districts)
+NUM_ORDERS = 3000
+STOCKS = 100000
+NUM_SUPPLIERS = 10000
+NUM_NATIONS = 62
+NUM_REGIONS = 5
+
+
+class TPCCLoader():
+    def __init__(self, dsn, w_id, scale_factor, start_date=None):
```

Comment:
What does `w_id` stand for? Could you please rename it to something more meaningful?

Reply:
done